src/unify/mod.rs

```diff
@@ -447,6 +447,31 @@ impl<S: UnificationStoreMut> UnificationTable<S> {
 /// ////////////////////////////////////////////////////////////////////////
 /// Public API
 
+impl<S, K, V> UnificationTable<S>
+where
+    S: UnificationStoreBase<Key = K, Value = V>,
+    K: UnifyKey<Value = V>,
+    V: UnifyValue,
+{
+    /// Attempts to return the current value for the given key. If the key has
+    /// been union'd, this may give the current value from the current root or `None`.
+    ///
+    /// This is useful for fast-paths that want to avoid pointer-chasing.
```

Comment:
```suggestion
    /// Obtains current value for key without any pointer chasing; may return `None` if key has been union'd.
```

How does this sound?